src/engineering-meeting/meeting-randomiser.ts

```diff
@@ -49,37 +66,37 @@ function allRequiredBlocks() {
    return allBlocks.filter(isRequiredBlock)
 }
 
-function isRequiredBlock(block: BuildingBlock) { 
-   return (block.isRequired); 
-} 
+function isRequiredBlock(block: BuildingBlock) {
+   return (block.isRequired);
+}
 
-function isNotRequiredBlock(block: BuildingBlock) { 
-   return (!block.isRequired); 
-} 
+function isNotRequiredBlock(block: BuildingBlock) {
+   return (!block.isRequired);
+}
 
 function getDateTime(): string {
    const date = new Date()
-   return "Date Generated: " + date.toLocaleString()
+   return date.toLocaleString()
 }
 
 class NameRandomiser {
-   private nouns = ['Russian Rocket', 'Egg Salad', 'Yale-鸭力', 'Ham Sandwich', 'Chicken McNugget', 'Club Turkey', 'Yellow Hat', 'Isolation', 'Wiggly Finger', 'Chi-Squared', 'Unicorn', 'Holographic', 'Extract',
```

Comment:
I would like to redefine completely these adjectives, nouns, terms. For the time being I am removing some weird stuff. 

What I have in mind is them to be more software-engineering oriented but also to include more nouns representing day-to-day things, like: Animals, Vegetables, Common objects...